.gitignore

```diff
@@ -27,3 +27,5 @@ coverage
 packages-hybrid/hybrid-mobile-actions/tests/TestProjects/*/theme/styles/native/
 packages-hybrid/hybrid-mobile-actions/tests/TestProjects/*/theme/settings-native.json
 packages-hybrid/hybrid-mobile-actions/tests/TestProjects/*/theme/styles.js
+
+detox/ios/NativeComponents.app
```

Comment:
sure this need to be ignored?

Reply:
Yes, if you run npm run test:ios it will decompress the app and this folder will appear as uncommited